cmd/rdma/main_test.go

```diff
@@ -66,8 +66,8 @@ func generateNetConfCmdAdd(netName string, cIfname string, deviceID string) rdma
 	}
 }
 
-func generateArgs(nsPath string, cid string, cIfname string, netconf rdmaTypes.RdmaNetConf) skel.CmdArgs {
-	bytes, _ := json.Marshal(netconf)
+func generateArgs(nsPath string, cid string, cIfname string, netconf *rdmaTypes.RdmaNetConf) skel.CmdArgs {
+	bytes, _ := json.Marshal(*netconf)
```

Comment:
Is it considered best practice to pass by value on Marshall and by reference on Unmarshall?

Reply:
so on unmarshal you have to because you need to fill the struct. Marshall seem to work with by value and by ref. 
gocritic didn't complain because it is defined as dynamic type (interface {}), but on generateArgs it complain that we pass by value because the struct is big.
[1]- https://goinbigdata.com/golang-pass-by-pointer-vs-pass-by-value/